runtime/web_worker.rs

```diff
@@ -630,11 +637,13 @@ impl WebWorker {
         v8::String::new(scope, &format!("{}", self.id))
           .unwrap()
           .into();
```

Comment:
Won't this cause a change of name (because we have a wrapper struct)?

Reply:
The `id_str` value will still be `worker-{id}` due to `fmt::Display` implementation or did I not understand your question?

Reply:
Ah, you're right